reactnative/resources

```diff
@@ -0,0 +1 @@
+../resources
```

Comment:
same question as for the `assets` folder.  Isn't this a symlink? and if so should it be committed?

Reply:
Yes, it's a symlink and it should be committed in order to work on React Native.